pydantic/_internal/_config.py

```diff
@@ -47,6 +47,7 @@ class ConfigWrapper:
     ignored_types: tuple[type, ...]
     allow_inf_nan: bool
     json_schema_extra: dict[str, object] | JsonSchemaExtraCallable | None
+    json_encoders: dict[type[object], JsonEncoder] | None
```

Comment:
you need to add this to the docstring above, perhaps including the warning.

Reply:
done